.gitignore

```diff
@@ -14,5 +14,7 @@ npm-debug.log*
 /backstop_data/ci_report/
 /backstop_data/backstop_scenarios.json
 /backstop_data/packageDependentMap.json
+backstop_data/engine_scripts/
+backstop.json
```

Comment:
I don't think we have to ignore these files.

- engine_scripts look like source code
- backstop.json isn't in the root directory

Do you have a backstop.json in your project root?

Reply:
I do, probably from trying to troubleshoot VRT issues. 

@jianliao can I delete backstop.json from my root? Might it have wound up there for a reason while trying to get VRT working? Should I be adding backstop_data/engine_scripts/ to my gitignore, or should that directory be deleted as well?

Reply:
Yeah, I think we should leave the `.gitignore` as it is, without these two additions. I think you're right @lunarfusion - they might have been due to your trying to troubleshoot VRT issues.

Reply:
New [commit](https://github.com/adobe/spectrum-css/pull/1488/commits/d8fa0d0ca97daedd4cdbe3465a95c37a85aa9acb) resolves this.